tasks/js-dom.md

```diff
@@ -10,6 +10,18 @@ Learn some frontend basics: DOM API
 - [JavaScript and the DOM](https://classroom.udacity.com/courses/ud117)
 (4 lessons, estimated completion time 8 hrs)
 - [freecodecamp Algorithm Scripting Challenges](https://learn.freecodecamp.org/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting) - finish second half of the course (10 lessons) and save a screenshot.
+- **Practical task**: Implement interactive side-menu' site without page reloads aka simple SPA. Theme may be any: pokemons, phones, site pages aka about/prices/goal/etc.
+Action menu - something like this but not so ugly :)
+![image](https://i.imgur.com/KcUYO4F.png)
+
+**‎Goals**:
+
+- [ ] complex implementation of full page responsive layout (credit to Udacity course about responsive HTML/CSS).
+- [ ] the point is the action-driven menu with ability to load some predefined html content related to own menu item into right-positioned `<main>` content container
```

Comment:
What is "action-driven menu"? I think it would be great if you provide some link to article or smth.

Reply:
haven't any idea about article :)